clusterman/interfaces/types.py

```diff
@@ -27,7 +27,6 @@ class AgentMetadata(NamedTuple):
 
 class InstanceMetadata(NamedTuple):
     market: InstanceMarket
-    weight: float
```

Comment:
Why aren't we tracking weight anymore?

Reply:
Originally I had swapped `weight: float` for `resources: ClustermanResources`, but then ended up removing resources from InstanceMetadata in favor of `total_resources` on AgentMetadata.

But weight is calculable from market anyway.